espresso/CP/h2o_32.in

```diff
@@ -0,0 +1,168 @@
+&CONTROL
+  title = "water 32 Molecules - PBE 330K",
+  calculation = 'cp-wf',
+  restart_mode = "from_scratch",
+  prefix = "h2o_32",
+  pseudo_dir = '.',
+  outdir = '.',
+  nstep = 1000,
```

Comment:
It takes quite a while to run this - could we reduce the number of steps for the purpose of this example? Perhaps we can also explain in README that this is not a "production" run, but just a calculation for demonstration purposes.

Reply:
100 stop would suffice